src/sceneParser.lua

```diff
@@ -40,9 +41,9 @@ function SceneParser.loadScene(sceneLines, world, location, inputs)
 	end
 
 	local function spawnObject(key, ship)
-		local shipID, location, data, shipInfo, shipType = unpack(ship)
+		local shipID, location, data, appendex = unpack(ship)
```

Comment:
Is this supposed to be `appendix`?

Reply:
What is `appendix`? Optional data?

Reply:
Appendix just something distinctively different than data and isn't used for most objects. Structure is currently the only one that uses it. I am still trying to figure out the specifics.